apps/evm/lib/evm/configuration.ex

```diff
@@ -115,8 +115,8 @@ defmodule EVM.Configuration do
       "Byzantium" ->
         EVM.Configuration.Byzantium.new()
 
-      "Constantinople" ->
-        EVM.Configuration.Constantinople.new()
+      # "Constantinople" ->
```

Comment:
Should this be commented out?

Reply:
@MattMSumner  currently, Constantinople tests are passing on the latest `develop` common tests branch. It should be commented out in your PR https://github.com/poanetwork/mana/pull/485